migrations/2023-12-19-114249_fastn_auth_remove_nullables/up.sql

```diff
@@ -0,0 +1,25 @@
+-- Your SQL goes here
```

Comment:
For now let's coalesce all migrations into a single migration. When this PR gets merged and the first release is created we should have only two migrations, one that diesel creates for its own purpose, and the other that will initialise all tables etc.

Reply:
Sure, I'll do that.

I was thinking of using [diesel_migrations](https://docs.rs/crate/diesel_migrations/2.0.0)'s `embed_migration!` to pack and run these migrations on `FASTN_DB_URL` if say `FASTN_ENABLE_AUTH=true`. What do you think?

Later when we have `fastn migration` subcommand, we can generate migrations for auth related tables in user's project (like django) if they've enabled the auth feature.

Reply:
Our migrations should be embedded in fastn itself. `fastn migrate` is a tool so end users can create their own tables and manage migrations. They are technically orthogonal, we can manage our tables using a different mechanism that how we manage end user's tables.

But I think diesel can take care of both. One set of migration will have to be embedded and other set discovered (and not just in top level package, but a all packages that have db access, how do we give db access is not yet decided).